zebrad/tests/common/configs/getblocktemplate-v1.0.0-rc.4.toml

```diff
@@ -0,0 +1,71 @@
+# Default configuration for zebrad.
```

Comment:
Copied from the test error message:
https://github.com/ZcashFoundation/zebra/actions/runs/3998037053/jobs/6860159622#step:14:3945

Reply:
I didn't get a chance to check what had changed in the format, but I can do it tomorrow if no-one else has time.